src/app/shared/services/signer.service.ts

```diff
@@ -191,7 +193,5 @@ export class SignerService {
 }
 
 interface LoginOpts {
-  email?: string;
-  wallet?: string;
   force?: boolean;
```

Comment:
Why have an interface for only one param?

Reply:
Every subsigner has its own login options and signer has its own options. When they are initialized, those two options are merged, so that's why it's better to keep it in the interface even though it's the only parameter. Moreover, we can add new options in anytime, so it will be easier to maintain.